hotelReservation/helm-chart/hotelreservation/charts/mongodb-rate/values.yaml

```diff
@@ -12,3 +12,7 @@ container:
   name: hotel-reserv-rate-mongo
   ports:
   - containerPort: 27017
+
+useAccessControl: true
+useInitScript: true
+initScriptConfigMap: mongo-rate-script
```

Comment:
Why is this a configuration variable?

Reply:
We use a different script for each service, so in order for the base deployment to use the right one it's a variable.

Reply:
I see that now in geo and rate.  Thanks.